packages/progress-bar/src/ProgressBar/ProgressBar.tsx

```diff
@@ -86,16 +125,34 @@ export function ProgressBar(props: ProgressBarProps) {
               theme,
               color,
               disabled,
-              isIndeterminate,
               value,
               maxValue,
+              animationMode,
             })}
+            // if on fade-out transition, revert back to base mode
+            onTransitionEnd={() => {
+              if (animationMode === AnimationMode.Transition) {
+                setAnimationMode(
+                  getAnimationMode({
+                    type,
+                    isIndeterminate,
+                    enableAnimation,
+                  }),
+                );
+              }
+            }}
           ></div>
         </div>
       </div>
 
       {description && (
-        <Description darkMode={darkMode} disabled={disabled}>
+        <Description
+          darkMode={darkMode}
```

Comment:
Just noticed this but if you wrap the component in the `LeafyGreenProvider`, then you don't have to pass `darkMode` to each child component. The component will read the value passed to the provider. Here's [an example from Callout](https://github.com/mongodb/leafygreen-ui/blob/main/packages/callout/src/Callout/Callout.tsx#L38).